src/igvfd/audit/file_set.py

```diff
@@ -536,3 +536,42 @@ def audit_MPRA_read_names(value, system):
             f'{unexpected_read_names} with `read_names` not associated with the MPRA uniform processing pipeline.'
         )
         yield AuditFailure(audit_message_unexpected.get('audit_category', ''), f'{detail} {audit_message_unexpected.get("audit_description", "")}', level=audit_message_unexpected.get('audit_level', ''))
+
+
+@audit_checker('MeasurementSet', frame='object')
+def audit_single_cell_read_names(value, system):
+    '''
+    [
+        {
+            "audit_description": "Sequence files are expected to have read names.",
+            "audit_category": "missing read names",
+            "audit_level": "ERROR"
```

Comment:
NOT_COMPLIANT, not ERROR

Reply:
missing properties are not ERROR since it is not incorrect metadata just incomplete.